### PR TITLE
Add suffix to notifier subject line if a partial order import success. 

### PR DESCRIPTION
**Pull request by williamsajr**

### CODE REVIEW

**Review by kdwyer — APPROVED** [review]
LGTM

File: addons/edi_notifier/models/edi_document.py [hunk 1]
```diff
@@ -21,6 +21,8 @@ class EdiDocumentType(models.Model):
 class EdiDocument(models.Model):
     _inherit = "edi.document"
 
+    notifier_subject_suffix = fields.Char("Suffix to the subject line of the notifier email")
```
Comment (PeterC10) [inline addons/edi_notifier/models/edi_document.py:24]: Nit: Should we capitalise the title - i.e. `Suffix to the Subject Line of the Notifier Email` ?

File: addons/edi_sale_notifier/models/edi_sale_request_document.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        SaleRequestRecord = self.sale_request_record_model(doc)
+
+        res = super().report_invalid_records(doc)
+        error_domain = [('doc_id', '=', doc.id), ('error', '!=', False)]
+        lines = SaleLineRequestRecord.search(error_domain)
+        orders = SaleRequestRecord.search(error_domain)
+        partners = PartnerRecord.search(error_domain)
+        if any([lines, orders, partners]):
+            doc.notifier_subject_suffix = " - with issues"
+        else:
```
Comment (PeterC10) [inline addons/edi_sale_notifier/models/edi_sale_request_document.py:27]: Is this needed? Won't `notifier_subject_suffix` be blank by default?

Comment (williamsajr) [reply addons/edi_sale_notifier/models/edi_sale_request_document.py:27]: I put the else as we want to clear any messages that may have been set. Perhaps on an earlier unsuccessful execution

**Review by PeterC10 — APPROVED** [review]
LGTM

File: addons/edi_notifier/models/edi_document.py (hunk 1 above)
Comment (williamsajr) [reply addons/edi_notifier/models/edi_document.py]: Changed

**Review by kdwyer — APPROVED** [review]

**Pull request merged**